docs/getting-started.md

```diff
@@ -0,0 +1,168 @@
+
+    ```python quick_start.py```
+
+    If the API client is installed and your authentication token is set, a CSV file called `gro_client_output.csv` should be created in the directory where the script was run.
+
+2. Try out [soybeans.py](https://github.com/gro-intelligence/api-client/blob/development/api/client/samples/crop_models/soybeans.py) to see the `CropModel` class and its `compute_crop_weighted_series()` method in action. In this example, NDVI ([normalized difference vegetation index](https://app.gro-intelligence.com/dictionary/items/321)) for provinces in Brazil is weighted against each province's historical soybean production to put the latest NDVI values into context. This information is put into a pandas dataframe, the description of which is printed to the console.
+
+    ```python crop_models/soybeans.py```
+
+3. See [brazil_soybeans.ipynb](https://github.com/gro-intelligence/api-client/blob/development/api/client/samples/crop_models/brazil_soybeans.ipynb) for a longer, more detailed demonstration of many of the API's capabilities in the form of a Jupyter notebook.
+
+4. You can also use the included gro_client tool as a quick way to request a single data series right on the command line. Try the following:
+
+    ```gro_client --metric="Production Quantity mass" --item="Corn" --region="United States" --user_email="email@example.com"```
+
+    The gro_client command line interface does a keyword search for the inputs and finds a random matching data series. It displays the data series it picked in the command line and writes the data points out to a file in the current directory called gro_client_output.csv. This tool is useful for simple queries, but anything more complex should be done using the Python packages.
+
+    Further documentation can be found in the api/client/ directory and on our wiki.
+
+## Discovering Data
+To derive the insights you want from Gro's data, you will first want to find the data you're interested in. Below you will find some of the most useful tips on how to discover the data of greatest value to you.
+
+### Code snippets
+Users may find that the Gro API is at its most powerful when used in conjunction with the Gro web application. The web app provides the most convenient format for selecting the data series that is of most interest to you. In our Add Data Series window, you can select entities of interest, and then other entities for which no data is available based on your selection will be filtered out from the remaining options. For example, after selecting the item "Corn" only metrics and regions that have data for 'Corn' will remain selectable.
+![add-data-series-example](../_images/add-data-series-example.png)

```

Comment:
For all of these missing images, I'm not quite sure why they don't work. Is it because they are in this PR and don't yet exist at the target location?

Reply:
I think the path should just not have an extra dot at the beginning:

```
![add-data-series-example.png](./_images/add-data-series-example.png)
```

Also it looks like your `.png` extensions are capitalized? I think it's case-insensitive, but not 100% sure.